src/main/java/tornadofx/Layouts.kt

```diff
@@ -28,6 +32,38 @@ fun GridPane.row(title: String? = null, op: (Pane.() -> Unit)? = null) {
     addRow(properties[GridPaneRowIdKey] as Int, *fake.children.toTypedArray())
 }
 
+fun GridPane.constraintsForColumn(columnIndex: Int): ColumnConstraints {
+    val constraints = columnConstraints
+    while (constraints.size <= columnIndex)
+        constraints.add(ColumnConstraints())
+    return constraints[columnIndex]
+}
+
+val Parent.columnConstraints: ColumnConstraints? get() {
```

Comment:
I switched it back and forth several times myself, ultimately preferring the shorter one on the grounds that I couldn't think of a way for the receiver to be ambiguous. I'm happy to leave it up to you.